muxrpc/handlers/tunnel/server/state.go

```diff
@@ -70,6 +74,23 @@ func (h *Handler) endpoints(_ context.Context, req *muxrpc.Request, snk *muxrpc.
 		return err
 	}
 
+	// cblgh:
+	// * reject if key in deny list / DeniedKeysService
```

Comment:
the `DeniedKeysService` lookup needs to be added to `roomsrv/init_network.go`, not here.

I agree that endpoints is a confusion name, it's really just _potentiall endpoints_ and tangentiall to the actual connection building.

Reply:
https://github.com/ssb-ngi-pointer/go-ssb-room/issues/89 :cat2:

Reply:
yeah i know, this is just a note to use it + remember what the service was called